### ISSUE TITLE
Provide dark theme icons

### ISSUE THREAD

**Issue by jakub-g** [rvagg/nodei.co#32]
What do you think about dark theme badges? (`&dark=true`) Basically replacing black color and gray background with black background and gray text. Would it be easy to implement it?

**Comment by rvagg** [comment]
Yeah, have a look through: https://github.com/rvagg/nodei.co/blob/master/lib/draw-npm-badge.js

First, all of those `rgb()` strings would need to be pulled up to consts and then a flag could be provide via the options that come in via the `draw()` function and be passed to all of the component drawing functions. Would be quite easy but what's the use-case? Where are you putting these badges that warrant this?

**Comment by jsejcksn** [comment]
GitHub dark theme is upcoming; it's already in the mobile apps.
